docs/front-end/sustainability.md

```diff
@@ -8,6 +8,6 @@ We use dark mode by default.
 
 All images should be renedered in the template file as webp format using the `format-webp` attribute - see https://docs.wagtail.org/en/v5.2.2/advanced_topics/images/image_file_formats.html.
 
-We have a custom saturation filter that allows us to apply a slight desaturation to all images using `saturation-0.6` - this makes the file size of images smaller overall, as well as creating a cohesive look for photographic images.
+We have a [custom saturation filter](https://github.com/torchbox/torchbox.com/blob/97547c3fc1cefa844d133e3af026c4df0dbca313/tbx/core/wagtail_hooks.py#L76-L90) that allows us to apply a slight desaturation to all images using `saturation-0.6` - this makes the file size of images smaller overall, as well as creating a cohesive look for photographic images.
```

Comment:
I think we need to remove this or add more explanation that we aren't using it currently.